eng/common/Extract-ReleaseNotes.ps1

```diff
@@ -1,65 +0,0 @@
-# given a CHANGELOG.md file, extract the relevant info we need to decorate a release
```

Comment:
This is a change under eng/common that would need to be made in azure-sdk-tools if you wish to delete this still.